config/cesm/machines/config_compilers.xml

```diff
@@ -1110,4 +1110,14 @@ using a fortran linker.
   <NETCDF_PATH>$ENV{NETCDF_DIR}</NETCDF_PATH>
 </compiler>
 
+<compiler COMPILER="gnu" MACH="modex">
+  <SLIBS>
+    <append> -L$ENV{HDF5_HOME}/lib -lhdf5_fortran -lhdf5 -lhdf5_hl -lhdf5hl_fortran </append>
+    <append> -lnetcdff -lnetcdf -lcurl -lblas -llapack</append>
```

Comment:
I think that this line still needs the -L$(NETCDF_PATH)

Reply:
In other words we need -L$(NETCDF_PATH) instead of -L$ENV{HDF5_HOME}/lib?

Reply:
Here is what my config_compilers.xml looks like within my clm-fates

```
<compiler COMPILER="gnu" MACH="modex">
  <NETCDF_PATH>$ENV{NETCDF_HOME}</NETCDF_PATH>
  <SLIBS>
    <append> -L$ENV{HDF5_HOME}/lib -lhdf5_fortran -lhdf5 -lhdf5_hl -lhdf5hl_fortran </append>
    <append> -L$ENV{NETCDF_HOME}/lib/ -lnetcdff -lnetcdf -lcurl -lblas -llapack</append>
  </SLIBS>
  <CPPDEFS>
     <append MODEL="gptl"> -DHAVE_VPRINTF -DHAVE_GETTIMEOFDAY -DHAVE_BACKTRACE </append>
  </CPPDEFS>
</compiler>
```

Reply:
Wait, never mind.  I see now that replacing HOME with PATH means we can get rid of the second line in compilers for NETCDF_HOME.....but what about the other options such as -lnetcdff -lnetcdf -lcurl -lblas -llapack?